src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(rustc_private, link_args)]
+extern crate rustc;
+extern crate syntax;
+extern crate byteorder;
+
+pub mod interp;
```

Comment:
Is the interp file missing from this PR?

Reply:
It is yes, partly deliberately to keep the first PR focused on the machine layout, but in hindsight I should have at least provided the `struct` being referenced.